utils/Database.py

```diff
@@ -28,6 +28,15 @@ def __repr__(self) -> str:
         return f"<User(email={self.email}, discord_id={self.discord_id}, todoist_id={self.todoist_id})>"
 
 
+class Score(Base):
+    __tablename__ = "scores"
+    email: Mapped[str] = mapped_column(primary_key=True)
+    streak: Mapped[int] = mapped_column()
+
+    def __repr__(self) -> str:
+        return f"<Score(email={self.email}, streak={self.streak})>"
+
+
```

Comment:
I don't hate this value just being added to the primary table, but if it is it's own table, it should follow ORM mapping: https://docs.sqlalchemy.org/en/20/orm/basic_relationships.html#one-to-one

Reply:
i'm still leaning for a seperate table, i think a category of points / score related things can exist outside of user logisitcs?

added one to one ORM mapping